packages/ui/src/components/Table/Cell/Cell.tsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react'
-import { Box } from '../../Box/Box'
+import { Box, BoxProps } from '../../Box/Box'
 
 import * as styles from './styles.css'
 
-export type Props = {
+export interface Props extends BoxProps {
```

Comment:
Nit, but what do you think about `Omit`ing the flex attributes that shouldn't be overridden?